client/src/styles/styles.css

```diff
@@ -0,0 +1,54 @@
+body {
+  margin: 0;
```

Comment:
I took the same approach zeroing out margin for the body to avoid any weird css the browser tries to add. Does it make sense to also do the same for padding?

Reply:
@juuprz didn't see padding getting affected. Was it for you? If so I can keep a watch for it when we combine components.